src/App.js

```diff
@@ -40,6 +43,9 @@ const App = () => (
     <Resource name='payments' {...payments} />
 
     <Resource name='student-grades' {...studentGrades} />
+    <Resource name='transcripts' {...transcripts} />
```

Comment:
C1. is it really necessary to have 3 different resource for adjacent path ?

we will talk about this later on